plugins/module_utils/redfish_utils.py

```diff
@@ -1899,6 +1900,9 @@ def multipath_http_push_update(self, update_opts):
             'UpdateParameters': {'content': json.dumps(payload), 'mime_type': 'application/json'},
             'UpdateFile': {'filename': image_file, 'content': image_payload, 'mime_type': 'application/octet-stream'}
         }
+        if image_type:
```

Comment:
There is no "image type" defined in the Redfish Specification. This looks like a very specific OEM definition for the OEM multipart form.

Would it be possible to make this more generic to accommodate other OEM usage? There's no predefined data type (it could be XML, JSON, binary, plain text...), and the form name isn't specifically "OemParameters".

Please check out Table 40 — Multipart HTTP push updates in the Redfish Spec here: https://www.dmtf.org/sites/default/files/standards/documents/DSP0266_1.20.2.pdf

Reply:
You can do `git revert <COMMITHASH>` to back out your changes from a given commit.

You then should make a new branch and cherry-pick that commit over.